src/wire/icmpv4.rs

```diff
@@ -339,6 +405,59 @@ impl<T: AsRef<[u8]> + AsMut<[u8]>> Packet<T> {
         NetworkEndian::write_u16(&mut data[field::ECHO_SEQNO], value)
     }
 
+    /// Set the identifier field (for timestamp request and reply packets).
+    ///
+    /// # Panics
+    /// This function may panic if this packet is not an timestamp request or reply packet.
+    #[inline]
+    pub fn set_timestamp_ident(&mut self, value: u16) {
```

Comment:
the `timestamp_ident` and `timestamp_seq_no` are exact copies of `echo_ident` and `echo_seq_no`, but I added them for consistency/readability.